benchmark/TestPerformance.ts

```diff
@@ -1340,7 +1340,7 @@ export class TestPerformance {
 						parser.removeErrorListeners();
 						if (!TestPerformance.TWO_STAGE_PARSING) {
 							parser.addErrorListener(DescriptiveErrorListener.INSTANCE);
-							parser.addErrorListener(new SummarizingDiagnosticErrorListener());
+							parser.addErrorListener(new SummarizingDiagnosticErrorListener() as ANTLRErrorListener<Token>);
```

Comment:
:memo: I'm not happy with needing to add the additional casts, so I'm planning to take a closer look with the goal of removing them.